app/pages/EditBreakingNewsPage.scss

```diff
@@ -0,0 +1,58 @@
+.edit-breaking-news-header, .edit-breaking-news-footer {
+    display: flex;
+    justify-content: center;
+}
+
+.edit-breaking-news-header {
+    font-size: 28px;
+    font-weight: 600;
+    padding: 10px 0px 0px;
+}
+
+.edit-breaking-news-footer {
+    padding-bottom: 20px;
+}
+
+.news-item-form {
+    margin: 0px 80px;
+    padding: 20px 0px;
+    h2 {
+        font-size: 20px;
+        font-weight: 600;
+    }
+    label {
+        font-size: 18px;
+    }
+}
+
+.news-item-form:not(:last-child) {
+    border-bottom: 1px solid grey;
+}
+
+.form-header, .form-header>button, .edit-breaking-news-footer>button, .form-section {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.form-section {
+    justify-content: space-between;
+    div:not(:last-child) {
+        padding-right: 20px;
+    }
+}
```

Comment:
This comment may no longer be applicable if you make the other changes I suggested regarding getting rid of the flexbox styling for the header and footer, but in general, I don't think it's a great idea to split styling definitions for the same class (`.form-section` in this specific case) across multiple blocks like this. It would be more OK to use multiple blocks if you were using different kinds of CSS properties in each, such as font styling in one and spacing in another.

However, in this particular case, you set `display: flex` in the first block but `justify-content` in the second, where `justify-content` only makes sense for flexbox or grid. This makes the CSS difficult to follow, especially if it's not immediately obvious that `.form-section` is also being defined elsewhere.

I think you should remove `.form-section` from the first block, and just compile define the `.form-section` in a single block.

Also, you probably do not want `align-items: center` for `.form-section`. This will cause the fields to line up strangely if they are of different sizes:

<img width="598" alt="Screen Shot 2023-02-20 at 11 46 55 PM" src="https://user-images.githubusercontent.com/1002748/220280674-24a35a8a-32c8-405b-8147-091996f372a1.png">

You probably really want `align-items: flex-end` so that the bottoms are all aligned.